### PR TITLE
MOBILE-2555: Upgrade to swift 3

### PR DESCRIPTION
**Pull request by brianblanchard-wf**
Description
---
- Upgrade w-mobile-kit as a dependency to wdesk-ios.

What Was Changed
---
- Updated all files to swift 3 syntax.
- Minor tweaks to auto complete view to use the new index pattern of swift 3

Testing
---
- [x] Ensure all unit tests pass
- [x] Play around with the sample app and see if you see any regressions
- [x] Test these changes on [MOBILE-2555](https://github.com/Workiva/wdesk-ios/tree/MOBILE-2555) in wdesk-ios.

---

Please Review: @Workiva/mobile

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Raven
Number of Findings: 0

**Comment by jeffscaturro-wf** [comment]
Not reviewing yet - but should this be a major version change? (MARV)

**Comment by jamesromo-wf** [comment]
Yes

**Comment by jamesromo-wf** [comment]
+10 to changes. I contributed to this work as well so we will need another reviewer.

**Comment by jamesromo-wf** [comment]
@jeffscaturro-wf 

Bug(?) Action sheet example - Portrait orientation - tap autosize sheet - rotate to landscape - it goes past the top of screen. I'm not sure if this is new or a regression.

Looks like that is existing. Basically we would need to trigger a redraw on rotation, which isn't super trivial so I'd vote to keep it as is until our app requires it or it's requested. I can reduce the size of the example so it's not as obvious.

File: Tests/WTestUtils.swift [hunk 1]
```diff
@@ -20,33 +20,35 @@ import Foundation
...
 
-    public override func locationInView(view: UIView?) -> CGPoint {
-        return returnPoint ?? CGPointZero
+    open override func location(in view: UIView?) -> CGPoint {
+        return returnPoint ?? CGPoint.zero
     }
 
-    public override func velocityInView(view: UIView?) -> CGPoint {
-        return returnVelocity ?? CGPointZero
+    open override func velocity(in view: UIView?) -> CGPoint {
```
Comment (jamesromo-wf) [inline Tests/WTestUtils.swift]: These are overrides so I believe they are apple functions we are matching.

**Comment by jamesromo-wf** [comment]
@jeffscaturro-wf For Bug(?) Switch and Radio Examples - Largest slider in middle seems to only trigger

Which device were you testing on? I haven't be able to reproduce that one yet.

**Comment by jeffscaturro-wf** [comment]
@jamesromo-wf my personal iPhone 7 plus

**Comment by jeffscaturro-wf** [comment]
+10 to latest, tickets will be made for the minor issues.

**Pull request merged**